<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MUIClass</class>
 <widget class="QMainWindow" name="MUIClass">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rcc/res.qrc">
    <normaloff>:/images/images/juk.png</normaloff>:/images/images/juk.png</iconset>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitterMain">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="libraryTreeView">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="indentation">
        <number>10</number>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTreeView" name="treeView">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DropOnly</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::CopyAction</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="indentation">
        <number>0</number>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="itemsExpandable">
        <bool>false</bool>
       </property>
       <property name="expandsOnDoubleClick">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Player Controls</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPrevious"/>
   <addaction name="actionPlay"/>
   <addaction name="actionNext"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAddMusicFiles"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPlaylist">
    <property name="title">
     <string>Playlist</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionErrorLog"/>
    <addaction name="actionEditStyleSheet"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlaylist"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../rcc/res.qrc">
     <normaloff>:/images/images/help.png</normaloff>:/images/images/help.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../rcc/res.qrc">
     <normaloff>:/images/images/exit.png</normaloff>:/images/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../rcc/res.qrc">
     <normaloff>:/images/images/open.png</normaloff>:/images/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="../rcc/res.qrc">
     <normaloff>:/images/images/clear.png</normaloff>:/images/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../rcc/res.qrc">
     <normaloff>:/images/images/save.png</normaloff>:/images/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="../rcc/res.qrc">
     <normaloff>:/images/images/media_playback_start.png</normaloff>:/images/images/media_playback_start.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="../rcc/res.qrc">
     <normaloff>:/images/images/next.png</normaloff>:/images/images/next.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="../rcc/res.qrc">
     <normaloff>:/images/images/previous.png</normaloff>:/images/images/previous.png</iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </action>
  <action name="actionErrorLog">
   <property name="text">
    <string>Error Log...</string>
   </property>
  </action>
  <action name="actionEditStyleSheet">
   <property name="text">
    <string>Edit Style Sheet...</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAddMusicFiles">
   <property name="text">
    <string>Add Music Files</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../rcc/res.qrc"/>
 </resources>
 <connections/>
</ui>
